components/reusable-components/footer/Footer.tsx

```diff
@@ -1,69 +1,173 @@
+                  formik={formik}
+                />
+                <button type="submit" className={`${styles.Button}`} disabled={formik.isSubmitting}>
+                  Submit
+                </button>
+              </div>
+            </div>
 
-        <div className={` title-l ${styles.socialMediaSection}`}>
-          <p>SOCIAL MEDIA</p>
-          <Link
-            href="https://www.linkedin.com/company/102600044/admin/feed/posts/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <i className="bi bi-linkedin" /> Linkedin
-          </Link>
-          <Link
-            href="https://www.facebook.com/profile.php?id=61556614894387"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <i className="bi bi-facebook" /> Facebook
-          </Link>
-          <Link
-            href="https://www.instagram.com/learnhub.mk?fbclid=IwAR3bWgALMMaxnp4a9hbUvjXLBOdkWGHcopCsFpp6WyKRRFoMSOyWqVHLNdk_aem_AUc6U-6f2fR5ErfT3g8nroxbZOsFEXjSAh6n06NlfDHWJxvbc-R-DNQ5hJhYnOLCuIWEVpD017qq83l-VqU5y43A"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <i className="bi bi-instagram" /> Instagram
-          </Link>
-          <Link
-            href="https://www.youtube.com/channel/UCDDywMbRvBXuUFTPBORc-BQ"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <i className="bi bi-youtube" /> Youtube
-          </Link>
-          <Link
-            href="https://discord.com/invite/nUEKUWVveW"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <i className="bi bi-discord" /> Discord
-          </Link>
-        </div>
+            <div className={`title-l ${styles.contactSection}`}>
+              <p>Contact us at</p>
+              <a href="mailto:contact@learnhub.mk">contact@learnhub.mk</a>
+            </div>
 
-        <div className={` title-l ${styles.newsletterSection}`}>
-          <p>NEWSLETTER</p>
-          <div className={`${styles.inputContainer}`}>
-            <EmailForm inputClassName={styles.newsletterSection} buttonClassName={styles.button} />
+            <div className={`title-l ${styles.socialMediaSection}`}>
+              <p>Connect with us</p>
+              <div className={styles.socialLinks}>
+                <Link
+                  href="https://discord.com/invite/nUEKUWVveW"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <Image src={discord} alt="Discord" width={48} height={48} />
+                </Link>
+                <Link
+                  href="https://www.linkedin.com/company/102600044/admin/feed/posts/"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <Image src={linkedin} alt="Linkedin" width={48} height={48} />
+                </Link>
+                <Link
```

Comment:
All this can be a separate component (meaning the icons with the names of the social media)
Maybe we will have a place to reuse it in the future